openmls/src/tree/tests_and_kats/unit_tests/test_resolution.rs

```diff
@@ -178,3 +182,66 @@ fn test_original_child_resolution() {
         );
     }
 }
+
+/// Test the `original_child_resolution` function that is used to calculate
+/// parent hashes
+#[test]
+fn test_exclusion_for_parent_nodes() {
```

Comment:
I don't a test in here. This is only doing tree operations. It either needs a test or a description of the test that is happening somewhere inside the test framework if this is the case.

Reply:
Good point. I added more explanations and changed the last `unwrap` to `expect`, which is where it will fail if the parent hash computation is incorrect.